android-app/app/src/main/java/de/dhbw/ka/se/fibo/CreateAccountActivity.java

```diff
@@ -55,9 +103,89 @@ private void initializeButton() {
     }
 
     private void registerButtonClicked() {
-        Intent i = new Intent(CreateAccountActivity.this,
+        i = new Intent(CreateAccountActivity.this,
                 MainActivity.class);
-        startActivity(i);
-        finish();
+        create_account_email = findViewById(R.id.create_account_email);
+        String email = create_account_email.getText().toString();
+        passwordField = findViewById(R.id.create_account_password_layer);
+        emailField = findViewById(R.id.create_account_email_layer);
+        create_account_password = findViewById(R.id.create_account_password);
+        String password = create_account_password.getText().toString();
+        createUser(email, password);
+
+    }
+
+    private void createUser(String email, String password) {
+
+        if (!checkValidInput(email, password)) {
+            return;
+        }
+
+        RequestQueue queue = Volley.newRequestQueue(this);
```

Comment:
```suggestion
        RequestQueue queue = FiBoVolley.getInstance(this).addToRequestQueue(stringRequest);;
```
**This is just a help and it will not work when commiting this suggestion**

We should implement this with a singleton pattern because in the future we will probably send multiple request across the network. 

In the [Volley documentation](https://google.github.io/volley/requestqueue.html#use-a-singleton-pattern) you can have a look on how your class should look like